packages/chakra-ui/src/SelectWidget/SelectWidget.tsx

```diff
@@ -78,42 +77,36 @@ const SelectWidget = (props: WidgetProps) => {
         </FormLabel>
       )}
       {typeof multiple !== "undefined" && enumOptions ? (
```

Comment:
Might I suggest refactoring this to a `const isMultiple = multiple !== "undefined" && Boolean(multiple);` and then you can simplify the if/else into a single render... Something like:
```
const isMultiple = multiple !== "undefined" && Boolean(multiple);
let formValue;
if (isMultiple) {
  formValue = (value || []).map((v: any) => ({
    label: _valueLabelMap[v],
    value: v,
  }));
} else {
  formValue = value || "";
}
...
       <Select
          inputId={id}
          name={id}
          isMulti={isMultiple}
          options={enumOptions as OptionsOrGroups<unknown, GroupBase<unknown>>}
          placeholder={placeholder}
          closeMenuOnSelect={!isMultiple}
          onBlur={_onBlur}
          onChange={isMultiple ? _onMultiChange : _onChange}
          onFocus={_onFocus}
          autoFocus={autofocus}
          value={formValue}
        />
...
```